secret-rotator/svckey/svckey.go

```diff
@@ -47,18 +47,24 @@ func (svc ServiceAccountKeySpec) Labels() map[string]string {
 }
 
 type Provisioner struct {
-	Service *iam.Service
+	// if EnableDeletion is set to true, the provisioner deletes the old svc key of 'version'
+	// when Deactivate('labels', 'version') is called.
+	EnableDeletion bool
+	Service        *iam.Service
 }
 
-func NewProvisioner() (*Provisioner, error) {
+func NewProvisioner(flag bool) (*Provisioner, error) {
```

Comment:
this variable name should be more descriptive and there should be a godoc comment on this exported method

Reply:
renamed to `enableDeletion`, and added godoc comments.